monad-logger/Control/Monad/Logger.hs

```diff
@@ -270,6 +282,29 @@ logError = logTH LevelError
 logOther :: Text -> Q Exp
 logOther = logTH . LevelOther
 
+
+-- | Generates a function that takes a 'Show a => a' and logs a 'LevelDebug' message. Usage:
+--
+-- > $(logDebug) "This is a debug log message"
```

Comment:
Can you add `@since` comments and do a minor version bump in the cabal file/changelog?